Sources/TabView.swift

```diff
@@ -489,7 +498,7 @@ extension TabView {
     private func moveTabItem(index: Int, animated: Bool) {
 
         switch options.addition {
-        case .underline:
+        case .underline, .circle:
             animateUnderlineView(index: index, animated: animated, completion: nil)
```

Comment:
Change the naming of this function as its meaning changes. (ex. `func animateAdditionView()` )